cmd/opdoc/opdoc.go

```diff
@@ -203,6 +247,8 @@ func opToMarkdown(out io.Writer, op *logic.OpSpec) (err error) {
 		assetParamsFieldsMarkdown(out)
 	} else if op.Name == "app_params_get" {
 		appParamsFieldsMarkdown(out)
+	} else if op.Name == "acct_params_get" {
```

Comment:
code style:
switch-case style branching would be a more elegant solution here

Reply:
Agree.  I'll change it.